### ISSUE TITLE
Failed to find Premain-Class manifest attribute

### ISSUE THREAD

**Issue by octobi** [vsilaev/tascalate-javaflow#3]
Instrumentation using the JavaAgent fails starting with Version `2.5.0`:
```
[error] Failed to find Premain-Class manifest attribute in [...]/.ivy2/cache/net.tascalate.javaflow/net.tascalate.javaflow.tools.javaagent/jars/net.tascalate.javaflow.tools.javaagent-2.5.0.jar
```

Compared to version `2.4.0`, version `2.5.0` jars manifest is missing `Premain-Class`, `Agent-Class`, `Can-Redefine-Classes`, `Can-Retransform-Classes`.

**Comment by vsilaev** [comment]
Confirmed.
A regression due to switch from Maven Assembly to Maven Shade plugin (affected both agents)
For now, there is a shaded version of Agent in [Tascalate JavaFlow Examples](https://github.com/vsilaev/tascalate-javaflow-examples)--
https://github.com/vsilaev/tascalate-javaflow-examples/blob/master/javaflow.instrument-cdi-proxy.jar
Probably it should works for you (just single argument to JVM "-javaagent:../javaflow.instrument-continuations.jar", no other run-time dependencies)

**Comment by vsilaev** [comment]
Fix committed to master, next release is planned till beginning of Mar

**Comment by octobi** [comment]
> Probably it should works for you (just single argument to JVM "-javaagent:../javaflow.instrument-continuations.jar", no other run-time dependencies)

I don't know how to specify this in the sbt enviromnent. I'm happy to help if someone can give me a quick hint:
```
  libraryDependencies ++= Seq(
    "net.tascalate.javaflow" % "net.tascalate.javaflow.api" % "2.5.0",
    "net.tascalate.javaflow" % "net.tascalate.javaflow.tools.javaagent" % "2.5.0"
  ),
  javaAgents += JavaAgent("net.tascalate.javaflow" % "net.tascalate.javaflow.tools.javaagent" % "2.5.0" % "compile;test;run;")
```
There's no urge for me on the topic, I have to resort to Java 8 due to other dependencies.

**Comment by vsilaev** [comment]
I don't know how to do it with SBT, but Java Agents (at least in this context) is a run-time only artifact, it should be not included in dependencies unless you are extending classes. What is necessary is to supply shaded agent JAR as -javaagent argument to JVM.

**Comment by vsilaev** [comment]
Fixed in 2.6.0

**Comment by octobi** [comment]
> Fixed in 2.6.0

I can confirm that.
